style.css

```diff
@@ -0,0 +1,319 @@
+html{
+    scroll-behavior: smooth;
+    scroll-padding: 3em;
+} 
+
+*{
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+    max-width: 100%;
+}
+
+body{
+    background-color:rgb(80, 102, 43);
+}
+
+.navbar{
+    height: 3em;
+    background-color: darkolivegreen;
+    display: flex;
+    align-items: center;
+    position: fixed;
+    width: 100%;
+    z-index: 9999
+}
+
+.nav-links{
+    width: 100%;
+    display: flex;
+    justify-content: space-around;
+}
+
+.nav-links a:hover{
+    color: rgb(250,250,180)
+}
+
+.nav-item{
+    list-style: none; 
+}
+
+a{
+    text-decoration: none;
+    color: lightgoldenrodyellow;
+    font-weight: bold;
+    letter-spacing: 3px;
+}
+
+p{
+    text-align: center;
+    font-size: 1em;
+    font-weight: bold;
+    letter-spacing: 1.5px;
+    word-wrap:break-word;
+    color: darkolivegreen;
+}
+
+.title{
+    text-align: center;
+    letter-spacing: 5px;
+    color: lightgoldenrodyellow;
+    margin-bottom: 40px;
+    font-size: 2em;
+}
+
+.sub-title{
+    text-align: center;
+    letter-spacing: 5px;
+    color: lightgoldenrodyellow;
+    margin-top: 150px;
+}
+
+.amex{
+    background-color: #006FCF;
+}
+
+#multiverse{
+    background-color: #4d61f4;
+}
+
+.home{
+    /* margin-top: 1em; */
+    height: 100vh;
+    background-color: darkolivegreen;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+
+.side-by-side{
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+}
+
+#about{
+    height: 100vh;
+    background-color: rgb(80, 102, 43);
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+.about{
+    /* height: 100vh; */
+    background-color: #4d61f4;
+    width: 60%;
+    padding: 5em;
+}
+
+#about-p{
+    text-align:start;
+    font-size: 1em;
+    font-weight: normal;
+    letter-spacing: 1.5px;
+    word-wrap:break-word;
+    color: lightgoldenrodyellow;
+}
+
+.technologies{
+    /* height: 100vh; */
+    padding: 5em;
+    background-color: rgb(80, 102, 43);
+    width: 40%;
+}
+
+.skills-p{
+    margin-top: 2em;
+    color: lightgoldenrodyellow;
+    letter-spacing: 3px;
+}
+
+li{
+    list-style: none;
+}
+
+.skills{
+    display: flex;
+    justify-content: space-around;
+}
+
+.skill-set{
+    margin-top: 1em;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+
+.skill{
+    margin-top: 2em
+}
+
+.front-end{
+    /* background-color: palegoldenrod; */
+    height: 50vh;
+    width: 45%;
+}
+
+.back-end{
+    /* background-color: palegoldenrod; */
+    height: 50vh;
+    width: 45%;
+}
+
+#soon{
+    color: rgb(250,250,210);
+    letter-spacing: 3px;
+}
+
```

Comment:
Thought:  I've noticed that you're using both classes and IDs with the same attribute name.  While there is nothing wrong with this, I wonder if this could be a potential bug-environment?  Would it be better to make it clear so that there would not be any silly misses between IDs and classes?